src/storage/compression/bitpacking.cpp

```diff
@@ -949,16 +949,42 @@ void BitpackingSkip(ColumnSegment &segment, ColumnScanState &state, idx_t skip_c
 	scan_state.Skip(segment, skip_count);
 }
 
+//===--------------------------------------------------------------------===//
+// GetSegmentInfo
+//===--------------------------------------------------------------------===//
+template <class T>
+string BitpackingGetSegmentInfo(ColumnSegment &segment) {
```

Comment:
Maybe it makes sense to turn this return type into a `InsertionOrderPreservingMap<string>` similar to  `PhysicalOperator::ParamsToString`?

Reply:
I guess `segment_info` becomes a `MAP(VARCHAR, VARCHAR)` then?
I do like that better for future-proofing yea

Reply:
Then `GetSegmentInfo` (for `String Uncompressed`) needs to change too. Alternatively, keep `segment_info` a string now but serialize it from the map?